cp-attendance-hybrid/src/data/mapper/subject/subject.mapper.ts

```diff
@@ -0,0 +1,59 @@
+import { ClassDay, Subject } from "@src/data/types/subjects/subject.type"
+
+export const subjectMapper = (subject): Subject => {
+  return {
+    name: subject.subject.name,
+    mainSubject: subject.subject.main_subject,
+    weekDateTimes: subject.regular_datetimes.map((date: any) => classDayMapper(date)),
+  }
+}
+
+export const mainSubjectMapper = (main_subject: string) => {
```

Comment:
Esse mapping parece estar se repetindo em vários lugares (mas pode ser impressão minha porque vi no outro PR)
Se tiver repetindo, eu condensaria num lugar só pra evitar erro de duplicata, mas se for noia minha só ignora